wagtailio/static/sass/components/_quotes.scss

```diff
@@ -0,0 +1,58 @@
+.quotes {
+    &__header {
+        color: $color--off-black;
+        padding-bottom: 60px;
+    }
+
+    &__author {
+        display: flex;
+        align-items: center;
+        padding: 30px 0 0 0;
+        color: $color--off-black
+    }
+
+    &__author-image{
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+        margin: 0 10px 0 0;
+    }
+
+    &__grid {
+        display: grid;
+        grid-gap: 40px;
+
+        @include media-query(medium) {
+            grid-template-columns: repeat(2, 40%);
+            grid-gap: 60px;
+            max-width: 80%;
+
+        }
+
+        @include media-query(large) {
+            grid-template-columns: repeat(2, 40%);
+            grid-gap: 80px 100px;
+            max-width: 80%;
+        }
+    }
+
+    &__icon {
+        fill: $color--teal-200;
+        position: absolute;
+        inset: -30px -55px auto auto;
+        width: 273px;
+        height: 184px;
+
+        @include media-query(medium) {
+            inset: -10% 10% auto auto;
+            width: 360px;
+            height: 450px;
+        }
+
+        @include media-query(large) {
+            inset: -15% 15% auto auto;
```

Comment:
In the designs the quote svg is higher up - slightly higher than the title:

<img width="750" alt="image" src="https://user-images.githubusercontent.com/31627284/181791528-9e386e63-43de-4b87-a9a1-7912182f6b76.png">

Reply:
No problem!